### PR TITLE
Add local time display to schedule

### PR DESCRIPTION
**Pull request by excitedleigh**
Here's what it looks like:

image

...and the mobile view:

image

### CODE REVIEW

File: src/Components/ScheduleItem/index.tsx [hunk 1]
```diff
@@ -9,6 +9,15 @@ export interface ScheduleItemProps {
   speaker: Speaker | null;
 }
 
+const UTC_FMT = new Intl.DateTimeFormat(
+  "en-GB-u-hc-h23",
```
Comment (patrick91) [inline src/Components/ScheduleItem/index.tsx:13]: is this supported everywhere? might be worth adding a fallback, not everyone is on recent browsers :)

File: src/Components/ScheduleItem/index.tsx [hunk 2]
```diff
@@ -64,6 +73,10 @@ export default class ScheduleItem extends React.Component<
 
     const SpeakerNameTag = isTalk && speaker!.social ? "a" : "p";
 
+    const [hourString, minString] = schedule.time.split(':', 2)
+    const time = new Date(Date.UTC(2020, 3, 25, parseInt(hourString), parseInt(minString)))
+    console.log(hourString, minString, time)
```
Comment (patrick91) [inline src/Components/ScheduleItem/index.tsx:78]:
```suggestion
```

**Comment by patrick91** [comment]
@excitedleigh thanks for the PR, that's going to be super useful. I've added a small note, let me know what you think and if that can be done quickly, otherwise I think we can merge 😊

File: src/Components/ScheduleItem/index.tsx (hunk 1 above)
Comment (excitedleigh) [reply src/Components/ScheduleItem/index.tsx:13]: It's reasonably widely supported, including IE 11: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat#Browser_compatibility

Comment (patrick91) [reply src/Components/ScheduleItem/index.tsx:13]: ok cool! let's get this in 😊 thanks a lot!

**Pull request merged**